akka-javasdk-tests/src/test/java/akkajavasdk/components/workflowentities/TransferWorkflowWithFraudDetection.java

```diff
@@ -86,11 +86,13 @@ public Effect<TransferState> getTransferState() {
     }
   }
 
+  // FIXME: use Effect.Transitional
   private Effect.TransitionalEffect<Void> finishWithSuccess(String response) {
```

Comment:
Here are a few examples of methods returning `Effect.TransitionalEffect<Void>`.

This is still working because a user that has such a method, can only use it when passing to `call(..).andThen(..)`. 

In this case, it's safe to introduce `Effect.Transitional` because old code will be using `call(..).andThen(..)`.